server/controllers/stock/reports/stock_value.report.handlebars

```diff
@@ -29,28 +29,32 @@
             <tr style="background-color:#ddd;">
               <th class="text-center">{{translate 'FORM.LABELS.CODE'}}</th>
               <th class="text-center">{{translate 'FORM.LABELS.INVENTORY'}}</th>
-              <th class="text-center">{{translate 'FORM.LABELS.INVENTORY_PRICE'}}</th>
               <th class="text-center">{{translate 'FORM.LABELS.QUANTITY'}}</th>
               <th class="text-center">{{translate 'STOCK.UNIT_COST'}}</th>
               <th class="text-center">{{translate 'FORM.LABELS.VALUE'}}</th>
+              <th class="text-center">{{translate 'FORM.LABELS.INVENTORY_PRICE'}}</th>
+              <th class="text-center">{{translate 'FORM.LABELS.SALE_VALUE'}}</th>
             </tr>
           </thead>
           <tbody>
             {{#each stockValues}}
-              <tr>
+            <tr {{#if hasWarning}}class="text-danger bg-danger"{{/if}} >
                 <td>{{inventory_code}}</td>
                 <td style="width:50%">{{inventory_name}}</td>
-                <td class="text-right">{{currency inventoryPrice ../currency_id 4}}</td>
                 <td class="text-right">{{stockQtyPurchased}}</td>
                 <td class="text-right">{{currency stockUnitCost ../currency_id 4}}</td>
                 <td class="text-right">{{currency stockValue ../currency_id}}</td>
+                <td class="text-right">{{currency inventoryPrice ../currency_id 4}}</td>
+                <td class="text-right">{{currency saleValue ../currency_id 4}}</td>
```

Comment:
Probably only need 2 fractional digits for this.

Reply:
For all of them, or just the `saleValue` total?

Reply:
Just the total sale value. The total value is already 2 digits.  The unit prices/cost should be 4 digits.

Reply:
Okay!  Thanks for the tip.  Will make the change.